### DISCUSSION TITLE
Colors problem

### DISCUSSION THREAD

**Discussion by 7iomka** [mantinedev/mantine#1772] [Q&A]
Hi.
I have button styles like this
<img width="429" alt="image" src="https://user-images.githubusercontent.com/18473137/178359777-941d48a8-08bc-4c82-aa5b-04e1733c3c44.png">

From docs I found for this purpose button variant like `subtle`.
But it uses under hood color for text from `primaryColor` settings from provider.
My question is how to override at global provider level this color for `subtle` variant of button to be inherited from body text color?

This is an example of a problem I'm having when setting a color that should be inherited and that isn't in the color palette (I mean for example white and black colors).
I need to make this change at the provider level because all variants which use text from primaryColor settings in my case need to change to `inherit`, because primaryColor in my case is too light.

**Comment by rtivital** [comment]
You can add styles to buttons and other elements with styles params on MantineProvider – https://v5.mantine.dev/theming/mantine-provider/#styles-on-mantineprovider
